instance/models/openedx_appserver.py

```diff
@@ -169,7 +169,16 @@ class Meta:
         help_text=(
             "Optional: A list of additional email addresses other than settings.ADMINS "
             "who should receive alerts from New Relic Synthetics Monitors when this instance "
-            "becomes unavailable, and alerts on AppServer provision failure."
+            "becomes unavailable."
+        )
+    )
+    provision_failed_emails = ArrayField(
```

Comment:
@swalladge, I am not a native English speaker, but isn't there ambiguity on whether this is for `provision "failed emails"` or `"provision failed" emails`? Would it be better to make it `provisioning_failed_emails` or something like `provisioning_failure_notifcation_emails`? This is just a suggestion and I will leave it to you to decide. :)

Reply:
Yep makes sense. I wasn't happy with the wording either :+1: